theano/gpuarray/opt.py

```diff
@@ -184,10 +193,11 @@ def local_opt(node):
                 # the context was derived from the outputs
                 for i in node.inputs:
                     i.tag.context_name = context_name
+
                 try:
-                    new_op = maker(node, context_name, node.inputs)
+                    new_op = maker(node.op, context_name, node.inputs)
                 except TypeError:
-                    new_op = maker(node, context_name)
+                    new_op = maker(node.op, context_name, node.inputs, out_clients)
```

Comment:
I would make out_clients like this:

out_clients = [o.clients for o in node.outputs]